tests/std/tests/P0898R3_concepts/test.cpp

```diff
@@ -1558,14 +1579,18 @@ namespace test_default_initializable {
         int x;
     };
     STATIC_ASSERT(default_initializable<S>);
+#ifdef __EDG__ // TRANSITION, VSO-1898941
+    STATIC_ASSERT(default_initializable<S const>);
+#else // ^^^ assert bug so we'll notice when it's fixed / no workaround vvv
     STATIC_ASSERT(!default_initializable<S const>);
+#endif // ^^^ no workaround ^^^
 
     // Also test GH-1603 "default_initializable accepts types that are not default-initializable"
-#if defined(__clang__) || defined(__EDG__) // TRANSITION, DevCom-1326684
+#if defined(__clang__) // TRANSITION, DevCom-1326684 (MSVC) and VSO-1898945 (EDG)

```

Comment:
No change requested: I _do_ like the annotations mapping bugs to compilers here so readers don't need to go to DevCom (or the internal database) and perform research.